### ISSUE TITLE
update branding of core and universe

### ISSUE THREAD

**Issue by xmfcx** [autowarefoundation/.github#2]
Under the https://www.autoware.org/media-kit , there is the [Brand Guide pdf](https://www.autoware.org/_files/ugd/a700a5_31b49b4418564de1a91b61afa60e8462.pdf) and it depicts the colors allowed for the `.IO`, `.AI`, `.AUTO` projects.

We don't even have the colors for these 2.

We will bring discussion of branding of `.CORE` and `.UNIVERSE`  in the next TSC meeting.

We will use the new branding to update the organizational profiles.

@mitsudome-r is it possible to facilitate this earlier?

**Comment by kenji-miyake** [comment]
FYI: Seeing https://www.autoware.org/autoware, Core and Universe are written as `Autoware Core` and `Autoware Universe`, no dot.

```
The future of Autoware is in Autoware Core and Autoware Universe.
```

> We will bring discussion of branding of .CORE and .UNIVERSE in the next TSC meeting.

:+1:

**Comment by ghost** [comment]
As a braindump of things that would be nice to have from a branding perspective:
- AWF banner image (eg: to add to the GitHub organization profile)
- Updated Autoware stack image
- Separate logos for Core and Universe 🤔 
- A more up-to-date version of this [RViz image](https://user-images.githubusercontent.com/10751153/158829692-e741e20f-912d-4788-b1ce-625c73f3a2cf.png), as it is over 2 years old
- an image version of the roadmap, or at least something that can be linked to directly from the organization profile page (the current roadmap on autoware.org is not an image and there's no way to link directly to that part of the https://www.autoware.org/autoware page)

**Comment by xmfcx** [comment]
## Icon suggestions

Following images are suggestions for what the icons could look like, not necessarily the icons themselves.

### Core
https://www.vectorstock.com/royalty-free-vector/black-hole-in-universe-symbol-with-spacetime-vector-20970835

![](https://cdn.vectorstock.com/i/1000x1000/08/35/black-hole-in-universe-symbol-with-spacetime-vector-20970835.webp)


### Universe
https://www.pixtastock.com/illustration/70058545

![](https://en.pimg.jp/070/058/545/1/70058545.jpg)

**Comment by ghost** [comment]
@xmfcx Those are both really cool (I love the core one). However, if we use royalty free images, isn't there a danger that someone else may use those same icons for something completely different?

**Comment by ghost** [comment]
Would it be useful if the organization profile page also briefly talks about Core and Universe?

**Comment by xmfcx** [comment]
> @xmfcx Those are both really cool (I love the core one). However, if we use royalty free images, isn't there a danger that someone else may use those same icons for something completely different?

Of course, I was trying to show that we could use those images as source of inspiration. I will add it to the post to make it clear!

**Comment by xmfcx** [comment]
> Would it be useful if the organization profile page also briefly talks about Core and Universe?

We could but It's already too long in my opinion. If we can shorten it, it could be possible.
